examples/kaleidoscope/main.rs

```diff
@@ -1193,16 +1193,15 @@ macro_rules! print_flush {
 // However, extern functions such as cos(x) and sin(x) can be imported without any problem.
 // Other lines related to this program can be found further down.
 
-// pub extern "C" fn putchard(x: f64) -> f64 {
-//     print_flush!("{}", x as u8 as char);
-//     x
-// }
+pub extern fn putchard(x: f64) -> f64 {
```

Comment:
These two extern functions are no longer used.. did you mean to remove them?

Reply:
Ugh, I tried to use them from the Kaleidoscope REPL, but didn't succeed, and forgot to comment them once again.